server/datastore/mysql/software.go

```diff
@@ -1145,7 +1112,6 @@ func (ds *Datastore) ListSoftwareVulnerabilities(
 		Select(
 			goqu.I("hs.host_id").As("host_id"),
 			goqu.I("cpe.software_id"),
-			goqu.C("cpe"),
```

Comment:
We don't need this anymore? Have we double checked with the interface team?

Reply:
This type is not exposed to the HTTP interface, the one that is still has the `cpe` value (https://github.com/fleetdm/fleet/blob/improving_vuln_detection/server/fleet/software.go#L58)